### ISSUE TITLE
Always defaults to Current Date

### ISSUE THREAD

**Issue by naveedahmed1** [IckleChris/ic-datepicker#4]
Hi,

The date field always defaults to current date, as a side effect, when I use it with reactive form and submit form without even selecting any date, it posts the current date to the server instead of null.

```
this.form = fb.group({
            from: [null],
            to: [null]
        });
```

**Comment by IckleChris** [comment]
Thanks @naveedahmed1, this has been fixed in `0.0.7`

**Comment by naveedahmed1** [comment]
Awesome! Thanks @IckleChris
